.env.template

```diff
@@ -18,8 +18,13 @@ FITBIT_POLL_INTERVAL_S=3600
 # 55001:  Spinning
 # 90013:  Walk
 # 90001:  Bike
+# 90019:  Treadmill
 # 1071:   Outdoor Bike
-FITBIT_ACTIVITY_TYPE_IDS=[55001, 90013]
+FITBIT_REALTIME_ACTIVITY_TYPE_IDS=[55001, 90013]
+FITBIT_DAILY_ACTIVITY_TYPE_IDS=[90019]
+FITBIT_DAILY_ACTIVITY_POST_TIME=23:50
```

Comment:
This is a breaking change: 
* Removed setting:
  - `FITBIT_ACTIVITY_TYPE_ID`
* Added settings:
  - `FITBIT_REALTIME_ACTIVITY_TYPE_IDS`
  - `FITBIT_DAILY_ACTIVITY_TYPE_IDS`
  - `FITBIT_DAILY_ACTIVITY_POST_TIME`